Framework/Algorithms/src/CorrectKiKf.cpp

```diff
@@ -191,31 +191,17 @@ void CorrectKiKf::execEvent() {
   g_log.information("Processing event workspace");
 
   const MatrixWorkspace_const_sptr matrixInputWS =
-      this->getProperty("InputWorkspace");
-  EventWorkspace_const_sptr inputWS =
+      getProperty("InputWorkspace");
+  auto inputWS =
```

Comment:
Did not notice that... same for `outputWS`. I think I will just remove both of the member variables, the seem pointless anyways and use local variables also in `exec()`.